docs/acc_activity_tutorial.rst.bak

```diff
@@ -0,0 +1,110 @@
+.. _acc_activity_tutorial:
```

Comment:
Looks like some "bak" files probably from your editor was included

Reply:
Thanks, I missed it in the excludes.